marimo/_server/rtc/doc.py

```diff
@@ -141,6 +141,10 @@ async def remove_client(
                 cleaner = self.loro_docs_cleaners.get(file_key, None)
                 if cleaner is not None:
                     cleaner.cancel()
+                    try:
+                        await cleaner
```

Comment:
Different fix from what you had before

Reply:
I did come up with a RC again after pulling this, but then wasn't able to replicate it. So better than previously, but the first fix maybe have been more robust for whatever reason

Reply:
ah you are right, reverting back